package.json

```diff
@@ -43,6 +43,7 @@
     "arb-bridge-peripherals": "^1.0.5"
   },
   "devDependencies": {
+    "@across-protocol/sdk-v2": "0.5.0-beta.16",
```

Comment:
Hm, why are we importing the SDK here? The SDK also depends on contracts-v2 and it looks like it's creating circular dependencies in `yarn.lock`.

Reply:
Removed